test/test.js

```diff
@@ -138,6 +138,28 @@ describe( 'rollup-plugin-babel', function () {
 		});
 	});
 
+	it( 'allows transform-runtime to inject builtin version of helpers', () => {
+		return bundle(
+			'samples/runtime-helpers-esm/main.js',
+			{ runtimeHelpers: true },
+			{},
+			{}
+		).then(({ code }) => {
+			assert.ok( !~code.indexOf( HELPERS ) );
```

Comment:
this assertion seems to be far from perfect, OTOH many other tests use this 🙈so 🤷‍♂️  for now, but it should get refactored later - maybe with jest snapshots? sounds like a good match

Reply:
Yeah, jest would be better here